travis/vars-nondocker.yaml

```diff
@@ -2,16 +2,13 @@ extends: vars_nondocker.yaml
 
 vars:
     checker:
-        defaults:
-            lang_files: [ngeo]
-            routes_disable: [printproxy_capabilities]
-        all:
-            routes_disable: [printproxy_capabilities]
-    check_collector:
-        disabled: [checker_pdf3, checker_fts]
```

Comment:
We should disable the checker_pdf3 check... => level 3 in this file?

Reply:
why do you want to disable it in non-docker mode?